app/components/participant-data.js

```diff
@@ -28,6 +21,9 @@ export default Ember.Component.extend({
                 this.set('reverse', false);
             }
             this.set('sortBy', sortBy);
+
+            const rawField = sortBy === 'modifiedOn' ? 'modified_on' : sortBy;
```

Comment:
If it makes any difference, I'm really sorry about this.

It turns out that sorting by `profileId` works as written, but `modifiedOn` fails to sort (it needs to be rewritten as `modified_on`).

There's probably some vagary of how we handle serializing field names in `meta` vs `attributes` that leads to this difference in behavior. But there's no transformation we can consistently apply for these two field names, so I just went with the narrow one-off on this page. A long term solution- consistent key names- is a bit outside the scope of this PR.